Providers/Scripts/2.6x-2.7x/Scripts/nxFile.py

```diff
@@ -999,6 +999,7 @@ def Get(DestinationPath, SourcePath, Ensure, Type, Force, Contents, Checksum, Re
         ModifiedDate = 0
         return [0, DestinationPath, SourcePath, Ensure, Type, Force, Contents, Checksum, Recurse, Links, Owner, Group, Mode, ModifiedDate]
 
+    Ensure = "present"
     stat_info = os.lstat(DestinationPath)
```

Comment:
what's the purpose of this change. is it only needed for python version 2.6x-2.7x ?